packages/react-native-bridge/ios/SourceFile.swift

```diff
@@ -35,7 +35,7 @@ extension SourceFile {
     
     public func jsScript(with argument: String? = nil) throws -> WKUserScript {
         let content = try getContent()
-        let formatted = String(format: content, argument ?? [])
+        let formatted = String(format: content, argument ?? "")
```

Comment:
I think it's appropriate to use `""` instead of `[]` here, given the `argument` type is `String`.

In fact, it may also result in a better behavior in case of `content` expecting an argument and none being provided at callsite:

<img width="557" alt="Screenshot 2023-04-12 at 12 05 30 pm" src="https://user-images.githubusercontent.com/1218433/231328956-9cdf7c40-197d-4131-8ecd-d86ce0400460.png">